### PR TITLE
clustering issues with develop branch fix

### PR DESCRIPTION
**Pull request by samyakjainnnnn**
Screenshot 2024-07-30 at 7 50 12 PM
1. issues with clustering in develop branch -Fixed!
2. Alignment fix

### CODE REVIEW

File: InnoTopicWebsite/src/styles.sass [hunk 1]
```diff
@@ -110,7 +110,7 @@ app-user-other-profile-icon
 
 .otherProfileIcon
```
Comment (JayTailor45) [inline InnoTopicWebsite/src/styles.sass]: @samyakjainnnnn Can you please move these css from `styles.sass`  to it's component?
Also if it is overriding the default component css than you can apply the style by selecting it's parent class.

**Review by JayTailor45 — CHANGES_REQUESTED** [review]

File: InnoTopicWebsite/src/app/cv-page/topics-graph/topics-graph.component.ts
Comment (JayTailor45) [inline InnoTopicWebsite/src/app/cv-page/topics-graph/topics-graph.component.ts:1]: @samyakjainnnnn Can you please refactor the code of this file in a way that your changes work only when the `graphHasContainer` flag is `true`, If the `graphHasContainer` flag is `false` then the code should work as it was working previously.

**Review by JayTailor45 — CHANGES_REQUESTED** [review]

File: InnoTopicWebsite/src/app/cv-page/topics-graph/topics-graph.component.ts
Comment (karol-depka) [reply InnoTopicWebsite/src/app/cv-page/topics-graph/topics-graph.component.ts:1]: graphHasContainer as router param please :)

**Review by karol-depka — CHANGES_REQUESTED** [review]

File: InnoTopicWebsite/src/app/cv-page/topics-graph/topics-graph.component.ts
Comment (samyakjainnnnn) [reply InnoTopicWebsite/src/app/cv-page/topics-graph/topics-graph.component.ts:1]: Updated!

File: InnoTopicWebsite/src/styles.sass (hunk 1 above)
Comment (samyakjainnnnn) [reply InnoTopicWebsite/src/styles.sass]: updated!

**Review by karol-depka — APPROVED** [review]
Thanks!

**Pull request merged**